logging/log_storage/installing-log-storage.adoc

```diff
@@ -26,6 +26,48 @@ include::modules/loki-create-object-storage-secret-console.adoc[leveloffset=+2]
 .Additional resources
 * xref:../../logging/log_storage/installing-log-storage.adoc#logging-loki-storage_installing-log-storage[Loki object storage]
 
+[id="installing-log-storage-loki-sts"]
+== Deploying a Loki log store on an STS enabled cluster
+
+For in cluster storage such as OpenShift Data Foundation or Minio, add the `credentialMode` field to your `Lokistack` CR with a value of `static` as shown in the following example.
+
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki # <1>
+  namespace: openshift-logging
+spec:
+  size: 1x.small # <2>
+  storage:
+    schemas:
+      - effectiveDate: '2023-10-15'
+        version: v13
+    secret:
+      name: logging-loki-s3 # <3>
+      type: s3 # <4>
+      credentialMode: static # <5>
+  storageClassName: <storage_class_name> # <6>
+  tenants:
+    mode: openshift-logging
+----
+<1> Use the name `logging-loki`.
+<2> Specify the deployment size. In the {logging} 5.8 and later versions, the supported size options for production instances of Loki are `1x.extra-small`, `1x.small`, or `1x.medium`.
+<3> Specify the secret used for your log storage.
+<4> Specify the corresponding storage type.
+<5> Represents the usage of static, long-lived credentials stored in a Secret. This is the default authentication mode and available for all supported object storage types.
+<6> Enter the name of a storage class for temporary storage. For best performance, specify a storage class that allocates block storage. Available storage classes for your cluster can be listed by using the `oc get storageclasses` command.
+
+For cloud based storage, you can use the xref:/../../authentication/managing_cloud_provider_credentials/cco-short-term-creds.adoc[Cloud Credential Operator (CCO)] and CCO utility (`ccoctl`) to configure authentication for your {logging} object store.
+
+[NOTE]
+====
+STS authentication must be configured during the installation of a new {product-title} cluster. You cannot configure an existing cluster that uses a different credentials strategy to use this feature.
```

Comment:
I think this should be 'new installation of Loki Operator' since we do not have STS mode on CLO. Also Customer upgrading to Loki 5.9 need to re-install Loki Operator to deploy it in managed/STS mode.